fixtures/generic/beam-275.json

```diff
@@ -0,0 +1,179 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "BEAM 275",
+  "shortName": "BEAM275",
+  "categories": ["Moving Head"],
+  "meta": {
+    "authors": ["vibeevents"],
+    "createDate": "2023-04-23",
+    "lastModifyDate": "2023-04-23"
+  },
+  "links": {
+    "other": [
+      "https://www.vibeevents.net"
+    ]
+  },
```

Comment:
Missing Manual link, Product page link, and video link.

The other link in here is to the users personal business website.